pom.xml

```diff
@@ -178,22 +177,6 @@
                 <artifactId>sonar-maven-plugin</artifactId>
                 <version>${sonar-maven-plugin.version}</version>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.8.0</version>
-                <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
-                    <annotationProcessorPaths>
-                        <path>
-                            <groupId>org.mapstruct</groupId>
```

Comment:
Maven compiler plug-in config no longer required as available by default. Even though there's specific config here for MapStruct, I don't think it was actually being used, based on unit tests still passing without it (on Java 8) and this link https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html.